testsuite/forge.py

```diff
@@ -1311,14 +1329,16 @@ def test(
     assert forge_cluster_name, "Forge cluster name is required"
 
     # cloud
-    if cloud.upper() == "AWS":
+    if cloud is not None:
```

Comment:
how can we tell if this feature is in use? (I think we need some kind of logging each invocation in humio!)

Reply:
We'll need to come up with a more comprehensive solution of how to log to Humio from Forge.